calcom/cal.com
# Fix booking page when toggling payment app settings

## What does this PR do?



Replaces getting `eventType.price` with data from `getPaymentAppData` in: 
- `BookingListItem`
- `EventTypeDescription`

Bookings with different payment options all under the same event type
<img width="831" alt="image" src="https://user-images.githubusercontent.com/65426560/233684385-a7fea6fe-4997-404c-b2bb-c408ce9b39c4.png">


Fixes # (issue)



**Environment**: Staging(main branch) / Production

## Type of change



- Bug fix (non-breaking change which fixes an issue)


## How should this be tested?


- Create an event type and install Stripe but do not enable
    - Create a booking
- Now enable Stripe and collect the booking on payment
    - Create another booking
- Now change to collect no-show fee
    - Create a booking
- Go to the  booking page
    - All bookings should appear as expected
- Disable Stripe on that event type and go back to the event type page. The price badge should be gone now

## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
